lighthouse-cli/bin.ts

```diff
@@ -114,6 +117,7 @@ Example: --output-path=./lighthouse-results.html`
   .default('output', Printer.GetValidOutputOptions()[Printer.OutputMode.pretty])
   .default('output-path', 'stdout')
   .default('port', 9222)
+  .default('timeout', Driver.MAX_WAIT_FOR_FULLY_LOADED / 1000)
```

Comment:
When it's a CLI interface for something whose order of magnitude is nearly always measured in seconds I've seen seconds more often than ms, but with a name change to more reflect that it's an internal code setting I think a move to ms is more reasonable.